### PR TITLE
Update data dictionary for Tasman and Nelson 2019 building outlines updates

### PR DESCRIPTION
**Pull request by jducnuigeen**
Fixes: #
Updated data dictionary to reflect changes in Tasman and Nelson areas.

### Change Description:
Data dictionary images updated to reflect additional building outlines coverage in Tasman and Nelson.
...

#### Source Code Documentation Tasks:
- [ ] README updated (where applicable)
- [x] CHANGELOG (Unreleased section) updated
- [ ] Docstrings / comments included to help explain code

#### User Documentation Tasks:
- [ ] Confluence user guide updated (where applicable)

#### Testing Tasks:
- [ ] Added tests that fail without this change
- [ ] All tests are passing in development environment
- [ ] Reviewers assigned
- [ ] Linked to main issue for ZenHub board

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -20,6 +20,7 @@ Fixed
 Changed
 -------
 * Some symbology changed to enable the removal of QgsHighlight due to bugs
+* Metadata updated to reflect additional Tasman and Nelson outlines.
```
Comment (dwsilk) [inline CHANGELOG.rst:23]: Data dictionary and metadata updated to reflect Tasman and Nelson building outlines updates.

File: db/docs/source/conf.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from builtins import str
+# from builtins import str
```
Comment (dwsilk) [inline db/docs/source/conf.py:1]: If this isn't required, remove the line?

Comment (jducnuigeen) [reply db/docs/source/conf.py]: Thanks. Done

File: CHANGELOG.rst (hunk 1 above)
Comment (jducnuigeen) [reply CHANGELOG.rst:23]: Done

**Pull request merged**